### PR TITLE
HiGlass Multivec + HiGlass Upgrades

### PR DESCRIPTION
**Pull request by utku-ozturk**
Trello: https://trello.com/c/nBNMlaxK

### CODE REVIEW

File: src/encoded/static/components/item-pages/HealthView.js [hunk 1]
```diff
@@ -235,18 +239,15 @@ export default class HealthView extends React.PureComponent {
...
             [ microMetaVersionUsed ] = microMetaFrom.split('#').splice(-1);
         }
-        //extend context to include vitessce version
-        const { dependencies } = installedPackageLockJson;
-        let vitessceVersion;
-        if (dependencies && dependencies.vitessce) {
-            vitessceVersion = dependencies.vitessce.version;
-        }
+        //extend context to include higlass & vitessce version
+        const { dependencies: { 'higlass': { version: higlassVersion } = {}, 'vitessce': { version: vitessceVersion } } } = installedPackageLockJson;
```
Comment (Bianca-Morris) [inline src/encoded/static/components/item-pages/HealthView.js:243]: Nice cleanup. I think we might do something like this for CGAP's health page, too, when I do this update.

**Review by Bianca-Morris — APPROVED** [review]
JS Code looks good to me. I did take a look at hotseat, but think @alexander-veit or @aschroed would probably know better if things are still displaying/working as they should. Just not familiar enough with this vis to really say either way.

**Review by aschroed — APPROVED** [review]
I've looked on hotseat and looks good to me.  Not sure if there are any multivec files there to check so did not check that.

**Comment by Bianca-Morris** [comment]
> I've looked on hotseat and looks good to me. Not sure if there are any multivec files there to check so did not check that.

It's probably a good idea to double check this. On CGAP's last Higlass upgrade, we uncovered a lot of Higlass issues that we thought were simply data-related after testing on an env with more data. If there's an easy/not-super-complicated way to do this, it might be worth it.

**Comment by aschroed** [comment]
> 

I found a multivec file on hotseat and looks good!
http://fourfront-hotseat-1650461284.us-east-1.elb.amazonaws.com/files-processed/4DNFINGT4SXC

**Pull request merged**